lib/model-interfaces/langchain/functions/request-handler/adapters/base/base.py

```diff
@@ -342,3 +362,245 @@ def run(self, prompt, workspace_id=None, *args, **kwargs):
                 return self.run_with_chain(prompt, workspace_id)
 
         raise ValueError(f"unknown mode {self._mode}")
+
+
+class BedrockChatAdapter(ModelAdapter):
```

Comment:
I think this is a copy of https://github.com/aws-samples/aws-genai-llm-chatbot/blob/main/lib/model-interfaces/langchain/functions/request-handler/adapters/bedrock/base.py

It would revert this change in this file.